### ISSUE TITLE
Metadata Manipulating Mods Aren't Applied Outside of Default Collection

### ISSUE THREAD

**Issue by DevJinnial** [xivdev/Penumbra#50]
I was doing some testing on the collections with metadata manipulating mods and I noticed something odd. For this example I was using the [Dragon Within](https://www.nexusmods.com/finalfantasy14/mods/473) (requires [The Body 2.5](https://www.xivmodarchive.com/modid/17279)) as well as the [Dragon Within No. 2 Type B Leggings](https://www.nexusmods.com/finalfantasy14/mods/514). I did some testing and the results are listed below. Note that in all tests `The Body 2.5 (Type 2)` is set to priority 0, `Dragon Within (Base and Type 2)` priority 1, and `No. 2 Leggings DW Version` priority 2.

## Example 1: Default Collection
Mod Names                                   | Default Collection | Forced Collection
------------------------------------- | -------------------- | ------------------- 
The Body 2.5 (Type 2)                    | ✅| ❌
Dragon Within (Base and Type 2)  | ✅ | ❌
No. 2 Leggings DW Version           | ✅ | ❌

In this case where this is are collections for characters (as is the character name is not in the list of collections), everything is working as intended. Only thing of note is the body mod loads in without any body hair

## Example 2: Forced Collection Only
Mod Names                                   | Default Collection | Forced Collection
------------------------------------- | -------------------- | -------------------
The Body 2.5 (Type 2)                    |❌| ✅
Dragon Within (Base and Type 2)  | ❌| ✅
No. 2 Leggings DW Version           | ❌| ✅

In this case where this is no collections for characters (as is the character name is not in the list of collections) but the mod is instead installed into the forced collection, the following is observed:

- `The Body 2.5` mod loads without body hair
- The `Dragon Within` mod itself does not appear under any circumstances
- The `No. 2 Leggings DW Version` does not load in and the vanilla model is shown

## Example 3: Character Collection
Mod Names                                   | Default Collection | Forced Collection | Character Collection
------------------------------------- | -------------------- | ------------------- | ---------------------- 
The Body 2.5 (Type 2)                    |❌| ✅ | ❌
Dragon Within (Base and Type 2)  | ❌| ❌| ✅
No. 2 Leggings DW Version           | ❌| ❌| ✅

This environment preforms identically to example 2 even when you place `The Body 2.5` into the character collection alongside the other mods.

## Example 4: TexTools Installation
When I installed everything via TexTools in the order listed in the tables above, the mods all seems to work in the exact same fashion as example 1. 

## Final Remarks
In all of these tests, from what I was able to gather it appears the issue is just with the metadata not being applied outside of the default collection. The body hair not being loaded is just unique to the dragon within mod so I'm not too concerned about that. But yeah. Hopefully all this information was help.

Sorry if all this was a lot, I just wanted to test out every possible situation I could think of to narrow the issue down for ya. 

So in conclusion, **metadata on mods does not work outside of the default collection**

**Comment by Ottermandias** [comment]
Were these three the only mods you were using? Which of those apply meta manipulations of which kind?

The warning for the Forced collection
![image](https://user-images.githubusercontent.com/70807659/130316805-274c22b8-1e12-4a1a-96bc-60782b134f29.png)
is there by design. It is simply not possible to load, for example, the single eqp file in two different versions because Forced as well as Character write to it. If anything else manipulates a metadata that goes into the same file, it will have priority over all Forced Meta changes.

**Comment by DevJinnial** [comment]
These where the only mods enabled for all those tests. I have other mods installed but none of them were enabled and none of them were meta manipulating ones. The goal was to just have the Dragon Within (which requires the body) on a specific character and the normal body mod (which doesn't touch metadata) on all others.

**Comment by Ottermandias** [comment]
It's possible that this is the general character-customization-reloading issue, where most customizations do not reload at all on a redraw if any character or npc in the zone has them.
It may also be a different problem, but if it is not, character collections will probably not be able to solve that problem at all.
As I've stated multiple times, they're a super hacky work-around :D
I'll look at the specific mods later when I find the time and check if I can find anything that I can fix.

**Comment by DevJinnial** [comment]
Ahhh gotcha. Thanks for the info. I kinda assumed the forced collection wouldn't work for them regardless but wanted to test on it anyways just to be safe.

**Comment by DevJinnial** [comment]
Going to close this out for now since my report at this point is very stale.
